source/MaterialXRenderGlsl/TextureBaker.cpp

```diff
@@ -482,67 +480,73 @@ BakedDocumentVec TextureBaker::createBakeDocuments(DocumentPtr doc, const FileSe
         genContext.registerSourceCodeSearchPath(path / "libraries");
     }
     genContext.getShaderGenerator().setColorManagementSystem(cms);
-    StringResolverPtr resolver = StringResolver::create();
-    StringVec renderablePaths = getRenderablePaths(doc);
 
-    BakedDocumentVec bakedDocuments;
-    for (const string& renderablePath : renderablePaths)
+    // Compute the material tag set.
+    StringVec materialTags = udimSet;
+    if (materialTags.empty())
+    {
+        materialTags.push_back(EMPTY_STRING);
+    }
+
+    if (_outputStream)
+    {
+        *_outputStream << std::endl << "Working on material: " << materialPath << std::endl;
```

Comment:
I like the idea of displaying this "processing" message at the start of each material bake, so that users can follow the progress of a multi-material bake command.  For the extra newline that separates the messages for each material, I would recommend placing it after the final "wrote baked document" message, rather than before the initial processing message.